spec/use_cases/administrator/check_if_valid_ip_spec.rb

```diff
@@ -0,0 +1,79 @@
+describe CheckIfValidIp do
+  subject { described_class.new }
+
+  context 'invalid' do
```

Comment:
Might be good to test a typical "mistyped and invalid IP" here - I'd expect IPAddr to cover it, but it'll make me sleep easier at night.

`let(:address) { '192.618.0.1' }`

Reply:
Done